rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -662,3 +812,181 @@ def prediction_states(
         self._choose_last_user_input(trackers_as_states, use_text_for_last_user_input)
 
         return trackers_as_states
+
+
+class IntentMaxHistoryTrackerFeaturizer(MaxHistoryTrackerFeaturizer):
+
+    LABEL_NAME = "intent"
+
+    def _convert_labels_to_ids(
+        self, trackers_as_intents: List[List[Text]], domain: Domain
+    ) -> np.ndarray:
+        """Convert a list of labels to an np.ndarray of label ids.
+        The number of rows is equal to `len(trackers_as_intents)`.
+        The number of columns is equal to the maximum number of labels
+        that any Labels item has. Rows are padded with -1 if not all Labels
+        items have the same number of labels.
+
+        Returns:
+            A 2d np.ndarray of label ids.
+        """
+
+        # store labels in numpy arrays so that it corresponds to np arrays
+        # of input features
+        label_ids = [
+            [domain.intents.index(intent) for intent in tracker_intents]
+            for tracker_intents in trackers_as_intents
+        ]
+
+        pad_val = -1
```

Comment:
Could you make this value a constant declared in `rasa/utils/tensorflow/constants.py`? It can then be reused by the policy and that would ensure that there is no mismatch between what's used as padding value in the tracker featurizer and what padding value is used in the policy.

Reply:
Sounds good! How does `LABEL_PAD_ID` sound to you?